BoyumFoosballStats/Pages/MatchHistory/Models/MatchHistoryViewModel.cs

```diff
@@ -36,6 +38,31 @@ public void HandlePlayerClicked(Player player)
         _navigationManager.NavigateTo("/PlayerDashboard/" + player.Id);
     }
 
+    public string GetTrueSkillChange(Player player, string? matchId)
```

Comment:
Can this go in the `PlayerAnalysisService` instead? Might be need for other stuff later

Reply:
I moved it!